pages/linux/runuser.md

```diff
@@ -0,0 +1,24 @@
+# runuser
+
+> Run commands as a specific user and group IDs.
+> Do not ask for password. Only useful when run as root user.
+
+- Run 'command' under substitute 'user':
```

Comment:
Perhaps this will read better?
```suggestion
- Run a command under a substitute user:
```

Reply:
`runuser` can run many commands or even some scripts, not only one command. Disagree.